pkg/ddl/job_scheduler.go

```diff
@@ -122,6 +123,25 @@ func (l *ownerListener) OnBecomeOwner() {
 	}
 	l.ddl.reorgCtx.setOwnerTS(time.Now().Unix())
 	l.scheduler.start()
+
+	if err := kv.RunInNewTxn(ctx, l.ddl.store, true, func(_ context.Context, txn kv.Transaction) error {
```

Comment:
non-owner node also need to init those variables, as DXF is enabled by default

maybe init inside `InitDistTaskLoop`